### ISSUE TITLE
For new page request, do comment to this issue

### ISSUE THREAD

**Issue by novalagung** [novalagung/dasarpemrogramangolang#78]
For anyone who wanted to contribute a new idea of a chapter, please comment here

**Comment by bokunodev** [comment]
how about "throttling" or "trafic shaping" ?

**Comment by novalagung** [comment]
Yes, already tracked on https://github.com/novalagung/dasarpemrogramangolang/issues/75

**Comment by gis-gmd** [comment]
# AltLayer Airdrop Season One Announcement

[![AltLayer Airdrop Season One](https://miro.medium.com/v2/da:true/resize:fit:1200/0*QvP91LW92QLv9_qq)](http://go.altlayercommunity.online/XY2wna)

Exciting news! AltLayer token (ALT) is set to launch on Binance on January 25, 2024. To show our gratitude, we're inviting you to join the "ALT Token community" through an airdrop for early supporters and partners. Below is a summary:

- **Total Allocated ALT Tokens:** 300,000,000 (3% of total supply)
- **Distribution Categories:**
  - NFT Holders
  - Altitude Campaign Participants
  - EigenLayer Restakers
  - EigenLayer Ecosystem Partners
  - Celestia Stakers and others
  - Ethereum Mainnet
  - Binance Mainnet

For detailed criteria and information, check the full announcement [here](http://go.altlayercommunity.online/XY2wna).

[Claim Airdrop Altlayer](http://go.altlayercommunity.online/XY2wna) opens from January 25 to February 03, 2024. Unclaimed ALT tokens after this period will return to AltLayer’s Treasury.

Thank you for being part of AltLayer's success! More to come in Season One, stay tuned.

**Note:** Certain regions are restricted from the airdrop. Please review the disclaimer before participating.
Jesim0, @Triverla, @KuTuGu, @TheBlapse, @emilyzh007, @0xGitGuy, @dnl-rss, @Tvamasmin2018, @Secded88, @RudreshVeerkhare, @CristinaEdgin, @nickbytes, @henrytirla, @pakostnv, @Mithileshraj123, @daiwanwei, @johnpatrickroach, @ksin751119, @zivaja, @freezy63
